src/main/scala/view/scalaFX/FXControllers/BaseAppController.scala

```diff
@@ -221,4 +221,11 @@ class BaseAppController(
     }
   }
 
+  /**
+   * Method that update clock's hand corresponding to the current phase and show the name of the current phase
+   * @param generationPhase
+   *   the current generation phase
+   */
+  private def updateClock(generationPhase: GenerationPhase): Unit = clockController.updateClock(generationPhase)
```

Comment:
non ha senso richiamare questo metodo, fai direttamente in updateView `clockController.updateClock(generationPhase)`